### PR TITLE
MAINT: Updated f-string method

### PR DESCRIPTION
**Pull request by cyai**
### Describe your change:

Updated the code with f-string methods wherever required for a better and cleaner understanding of the code.

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by cclauss** [comment]
Please flynt the entire repo https://pypi.org/project/flynt/

**Comment by cyai** [comment]
Updated the entire repo

**Comment by cclauss** [comment]
Please fix the failing test when you get a chance.

File: ciphers/rsa_key_generator.py [hunk 1]
```diff
@@ -34,8 +34,8 @@ def generateKey(keySize: int) -> tuple[tuple[int, int], tuple[int, int]]:
 
 
 def makeKeyFiles(name: str, keySize: int) -> None:
-    if os.path.exists(f"{name}_pubkey.txt") or os.path.exists(
-        f"{name}_privkey.txt"
+    if os.path.exists("%s_pubkey.txt" % (name)) or os.path.exists(
+        "%s_privkey.txt" % (name)
     ):
```
Comment (cclauss) [inline ciphers/rsa_key_generator.py:39]:
```suggestion
    if os.path.exists(f"{name}_pubkey.txt") or os.path.exists(f"{name}_privkey.txt"):
```
I think black is saying that all fits into 88 characters per line.

Comment (cyai) [reply ciphers/rsa_key_generator.py]: Oh, thank you. Resolved do check

File: ciphers/elgamal_key_generator.py [hunk 2]
```diff
@@ -38,8 +38,8 @@ def generate_key(key_size: int) -> tuple[tuple[int, int, int, int], tuple[int, i
 
 
 def make_key_files(name: str, keySize: int) -> None:
-    if os.path.exists("%s_pubkey.txt" % name) or os.path.exists(
-        "%s_privkey.txt" % name
+    if os.path.exists(f"{name}_pubkey.txt") or os.path.exists(
+        f"{name}_privkey.txt"
     ):
```
Comment (cclauss) [inline ciphers/elgamal_key_generator.py:43]:
```suggestion
    if os.path.exists(f"{name}_pubkey.txt") or os.path.exists(f"{name}_privkey.txt"):
```

**Review by cclauss — APPROVED** [review]
Thanks for doing this!

**Pull request merged**